build/NuSpecs/UmbracoCms.SqlCe.nuspec

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+    <metadata minClientVersion="4.1.0">
+        <id>UmbracoCms.SqlCe</id>
+        <version>9.0.0</version>
+        <title>Umbraco Cms Core Binaries</title>
```

Comment:
The Title needs to incode something about SqlCE

Reply:
Thought about it but I saw that on the other nuspec files that we had previously we kept the same title, but can definitely change that!

Reply:
I think it is important to change this one, because this is an add-on to the rest of Umbraco. Not a necessary part.

Reply:
Makes sense!